docs/contributing/doing.md

```diff
@@ -0,0 +1,141 @@
+---
+id: doing
+title: Contributing
+sidebar_label: Working
+---
+
+# Working
```

Comment:
This is creating two h1 tags on the page, "Contributing" and "Working" we should only have one see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#Usage_notes

Reply:
Yes, though one of these is inside the <header> element of an <article>.  Either way, when we have a design we should be able to tweak the HTML template